collectors/python.d.plugin/python_modules/bases/FrameworkServices/UrlService.py

```diff
@@ -6,7 +6,7 @@
 
 import urllib3
 
-from distutils.version import StrictVersion as version
+from packaging.version import parse as version
```

Comment:
Is it the standard library? Looks like it is not.

```python
~ $ python3
Python 3.11.6 (main, Oct  2 2023, 13:45:54) [Clang 15.0.0 (clang-1500.0.40.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from packaging.version import parse as version
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'packaging'
```

Reply:
We could just remove it completely.

We use it to check the `urllib3` version, and we require a version >1.9. Considering this was released in 2014, we could assume by now we should be safe...

Reply:
Yes, I see 2 options:
 - remove it
 - copy StrictVersion from distutils to python_modules/third-party or something.